### ISSUE TITLE
not able to specify microseconds in the format field 

### ISSUE THREAD

**Issue by m4ccbr** [Juniper/open-nti#187]
Hi

what should I use in the format field to be able to display the yaxe in microseconds?

Al

**Comment by 3fr61n** [comment]
Hi

Are you talking about this parameter?


<img width="447" alt="screen shot 2017-09-26 at 12 08 25" src="https://user-images.githubusercontent.com/6707509/30854946-870658ee-a2b3-11e7-8fbe-96bbf9cc504b.png">

**Comment by 3fr61n** [comment]
If so... inside the dashboard json file, the yaxes block should looks like

```
         "yaxes": [
            {
              "format": "µs",    <--- Left Y
              "logBase": 1,
              "max": null,
              "min": null,
              "show": true
            },
            {
              "format": "short",  <-- Right Y
              "logBase": 1,
              "max": null,
              "min": null,
              "show": true
            }
          ],
```

**Comment by m4ccbr** [comment]
yep, but when I try to define "µs" in the dashboard template it does not work.

**Comment by m4ccbr** [comment]
sorry for the delay, I got disturbed by a lot of issues in the past few days :(

but that's what I see when I set µs

 opennti - INFO - Dashboard File name : data_collection_agent_dashboard.json
Traceback (most recent call last):
  File "gendashboard.py", line 182, in <module>
    text_file.write(dashboard_tpl_rdr)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xb5' in position 45648: ordinal not in range(128)


template: graph-lines-02.j2
datasource: influxdb
title: "RPM Probes"
span: 12
fill: 1

yaxes:
  - format: µs
  - format: short

targets:
  A:
    query: >
      SELECT mean(value)
      FROM /($host_regex).rpm.*.max-delay/
      WHERE $timeFilter
      GROUP BY time($GroupBy)
  B:
    query: >
      SELECT mean(value)
      FROM /($host_regex).rpm.*.min-delay/
      WHERE $timeFilter
      GROUP BY time($GroupBy)

templatings_used:
  - host_regex.yaml
  - group_by.yaml
